dev/forbidden_from_release_tests/pubspec.yaml

```diff
@@ -10,10 +10,10 @@ dependencies:
   package_config: 2.1.0
   path: 1.8.3
   process: 4.2.4
-  vm_snapshot_analysis: 0.7.2
+  vm_snapshot_analysis: 0.7.6
```

Comment:
Given this is bumping from 0.7.2 -> 0.7.6, I'm guessing it's picking up other changes than the fix? Can you publish a special hotfix release of `vm_snapshot_analysis`, like a version `0.7.2+1`, that way we can minimize the diff in the Flutter SDK?

I'll be writing up a pub dependency cherrypick to Flutter SDK playbook so that hopefully in the future there will be a clear steps to do this.

Reply:
All the non-dev versions from 0.7.2 to 0.7.6 were to address the null check failures in 0.7.2, so 0.7.2+1 would basically be the exact same package.  I can do that if desired, but it seems a bit silly to do so.

Reply:
ahh, ok, this sounds good to me then